docs/guide/examplegen.md

```diff
@@ -42,7 +42,7 @@ instead of `base_executor`. So following the Avro or Parquet example in the
 Executor source code may be advisable.
 
 In addition, these data sources and formats are available as
-[custom component](/tfx/guide/understanding_custom_components) examples:
```

Comment:
This is actually not a broken link on tensorflow.org.  It links to:

[`https://www.tensorflow.org/tfx/guide/understanding_custom_components`](https://www.tensorflow.org/tfx/guide/understanding_custom_components)